### ISSUE TITLE
waveform indexer "poll" parameter is buggy?

### ISSUE THREAD

**Issue by megies** [krischer/jane#65]
It looks like the `poll` parameter is making the indexer wait for the specified amount of seconds after every single directory that is scanned? Intuitively I would have thought that the poll time is applied after a full run across the whole directory tree..

**Comment by krischer** [comment]
I'm gonna offload this to @barsch as this is the old indexer from seishub ;-)

The `poll` parameter seems to be passed as the `timeout` parameter to `select.select()`.

**Comment by barsch** [comment]
no, thats by purpose - this was a requested option by Jens Oeser to limit extensive access to the network drives - so after each operation we wait for x seconds 

in ObsPy it is set to 0.1s by default (https://github.com/obspy/obspy/blob/master/obspy/db/scripts/indexer.py#L205), in Jane it is set to 0s (https://github.com/krischer/jane/blob/master/src/jane/waveforms/management/commands/index_waveforms.py#L430)

there is no such options (yet) to pause between full runs/directories ...

**Comment by megies** [comment]
OK, thanks for clarification. I think this should be clarified in the docs, eventually.
